### PR TITLE
docker: tooling update for container image

### PR DESCRIPTION
**Pull request by borkmann**
- Update iproute2 repo to pull in AF_ALG dependency removal (Fixes: #5112)
- Include bpftool for debugging / introspection

This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5114)

### CODE REVIEW

**Review by ianvernon — APPROVED** [review]

File: contrib/packaging/docker/Dockerfile.runtime [hunk 1]
```diff
@@ -7,7 +7,7 @@ apt-get upgrade -y && \
 #
```
Comment (ianvernon) [inline contrib/packaging/docker/Dockerfile.runtime:7]: You will have to update the `Dockerfile` to use the new image that you push, and then trigger a build on this PR with the new version to see if it functions correctly.

**Review by ianvernon — COMMENTED** [review]
Removing too-early approval (this is still wip)

**Comment by borkmann** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Review by eloycoto — APPROVED** [review]

**Comment by eloycoto** [comment]
@borkmann should we updated our CI base image to use that version too?

**Comment by borkmann** [comment]
The base image that would build Cilium itself would not need it, only the runtime that compiles and loads the BPF programs.

**Pull request merged**